ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -xeuo pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+  export IBMCLOUD_CLI=ibmcloud
+  export IBMCLOUD_HOME=/output
+  region="${LEASED_RESOURCE}"
+  export region
+  echo "Try to login..."
+  "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+ibmcloud_login
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+
+# Get existing vnet info
+VPC_CONFIG="${SHARED_DIR}/customer_vpc_subnets.yaml"
+if [[ ! -f "${VPC_CONFIG}" ]]; then
+  echo "Fail to find of VPC info file ${VPC_CONFIG}, abort." && exit 1
+fi
+echo "Reading variables from ${VPC_CONFIG}..."
+vpcName=$(yq-go r "${VPC_CONFIG}" 'platform.ibmcloud.vpcName')
+resource_group=$(yq-go r "${VPC_CONFIG}" 'platform.ibmcloud.resourceGroupName')
+infra_id=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+echo "Using region: ${region}  resource_group: ${resource_group} vpc: ${vpcName}"
+
+${IBMCLOUD_CLI} target -g ${resource_group} -r ${region}
```

Comment:
region is already set in `ibmcloud_login`, so I think we do not need the `-r ${region}` here.